asm/battle_tower.s

```diff
@@ -1340,7 +1340,7 @@ GetFrontierTrainerFrontSpriteId: @ 8162AA0
 	lsls r0, 1
 	cmp r4, r0
 	bne _08162AD0
-	ldr r1, =gTrainerClassToFrontPicId
+	ldr r1, =gUnknown_0831F578
```

Comment:
No need to call this a gUnknown again, it seems like I misunderstood what this table was for and it should be renamed back to something like gTrainerClassToFrontPicIdMap for clarity.